### ISSUE TITLE
Inline links - visited styles

### ISSUE THREAD

**Issue by jtart** [bbc/simorgh#425]
**Is your feature request related to a problem? Please describe.**
Visited styles of inline links do not meet UX requirements.

**Describe the solution you'd like**
Inline links should have the following styles in a visited state:
- border-bottom: 1px solid `#5C5752` pebble
- color: `#5C5752` pebble

background color: not applicable

**Additional context**
- This issue may be worked on at the same time as any PR in the following group: #424, #425, #426, #427

**Testing**
- Force visited state
- Cypress test
- Check superscript/subscript characters in links (dependent on #454)

**Screenshot of designs**
![image](https://user-images.githubusercontent.com/7791726/43959325-507dba72-9ca6-11e8-8212-69ea6fc75b16.png)

**Comment by pjlee11** [comment]
<img width="277" alt="inline link" src="https://user-images.githubusercontent.com/7791726/43958883-d978305c-9ca4-11e8-81c8-c540878d3665.png">

**Comment by nataliesmart** [comment]
Should be as follows:
color: #5C5752 pebble
border-bottom-color: 1px solid #5C5752 pebble
background colour: none
border: none
